app/graphql/resolvers/rdf.ts

```diff
@@ -157,7 +159,25 @@ export const dataCubesComponents: NonNullable<
 
   return (
     await getCubesDimensions(cubes, { locale, sparqlClient, filters, cache })
-  ).map((d) => ({ iri: d.data.iri }));
+  ).map(async (dimension) => {
+    const { cube, data } = dimension;
+
+    return {
+      __typename: resolveComponentType(dimension),
```

Comment:
Previously this was automatically done by the resolver, but now we have to put it manually right, because we tried to shorten the resolver chain, is this right ?

Reply:
Yes 👍